clang/include/clang/Basic/LangOptions.h

```diff
@@ -346,6 +346,8 @@ class LangOptionsBase {
     BKey
   };
 
+  using PointerAuthenticationMode = ::clang::PointerAuthenticationMode;
```

Comment:
What does this `using` directive serve for? I've deleted it locally and the build still succeeds.

Reply:
I deleted it as it's not needed for this PR.